frontend/libs/studio-components/src/components/StudioContentMenu/StudioMenuTab/StudioMenuTabContainer.module.css

```diff
@@ -0,0 +1,39 @@
+:root {
+  --selected-border-marking: 3px;
+}
+
+.tabIsSelected,
+.tab {
+  display: flex;
+  align-items: center;
+  gap: var(--fds-spacing-2);
+  border: solid 1px var(--fds-semantic-border-action-second-subtle);
+  padding: var(--fds-spacing-3);
+  cursor: pointer;
+}
+
+.linkIcon {
+  color: var(--fds-semantic-text-neutral-default);
+}
+
+.linkIcon,
+.icon {
+  display: flex;
+  align-items: center;
+  font-size: var(--fds-spacing-8);
+}
+
+.tabTitle {
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.tabIsSelected {
+  border-left: var(--selected-border-marking) solid var(--semantic-surface-action-default);
+  background-color: white;
+}
+
+.tab:hover,
```

Comment:
I can see that we have added css for when the tab is hovered, but should we also add css for when the tab is accessed by keyboard? 
Right now it looks like this: 
![image](https://github.com/user-attachments/assets/70da17d9-8045-4574-8015-c8fbdc8465c7)

I also checked how it looks like today in the SettingsModal, and I can see that the style added one year ago is not there anymore 😅 It was supposed to be the same yellow border as Designsystemet uses. Maybe we can add that to the tab? 
Example of another button with the yellow: 
![image](https://github.com/user-attachments/assets/b59dafbe-9079-4c7e-aef1-ecb745dc1b47)

Reply:
Yes, we can! I also checked the style in todays settingsmodal and figures it was intentional but I can fix. But I believe @Annikenkbrathen a black border should be used here? 🤔

Reply:
It now looks like this when I tested. Great job @standeren 😄 
![image](https://github.com/user-attachments/assets/65bccb33-537d-4dc4-aa01-2496659430e0)